Dockerfile

```diff
@@ -19,4 +19,4 @@ RUN npm ci --omit=dev
 COPY --from=builder /app/dist ./dist
 
 EXPOSE 3000
-CMD ["node", "dist/index.js"]
+CMD ["node", "--max-old-space-size=3072", "dist/index.js"]
```

Comment:
would we want leave this a little under 3072 (about 75%)?

[from node docs for garbage collection memory](https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes)

Reply:
Good catch. Done!